### PR TITLE
Allow additional fetch opts for GFI per query layer

### PR DESCRIPTION
**Pull request by dnlkoch**
## Description

This introduces the prop `fetchOpts` for the `CoordinateInfo` component to set additional request options that will be applied to the GetFeatureInfo request. The options can be set for each `queryLayer` separately and will be identified by the internal layer id.

Please review @terrestris/devs.

## Related issues or pull requests

--

## Pull request type

- [ ] Bugfix
- [x] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the
  [BSD-2-Clause](https://github.com/terrestris/react-geo/blob/main/LICENSE).
- [x] I have followed the [guidelines for contributing](https://github.com/terrestris/react-geo/blob/main/CONTRIBUTING.md).
- [x] The proposed change fits to the content of the [Code of Conduct](https://github.com/terrestris/react-geo/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally).
- [ ] I have added a screenshot/screencast to illustrate the visual output of my update.

### CODE REVIEW

File: src/CoordinateInfo/CoordinateInfo.tsx [hunk 1]
```diff
@@ -50,6 +51,13 @@ export interface CoordinateInfoProps {
    * The ol map.
    */
   map: OlMap;
+  /**
+   * Optional request options to apply (separated by each query layer, identified
+   * its internal ol id).
```
Comment (LukasLohoff) [inline src/CoordinateInfo/CoordinateInfo.tsx:56]:
```suggestion
   * by its internal ol id).
```

**Review by LukasLohoff — APPROVED** [review]
Looks good! Left one comment on a minor documentation issue

**Comment by dnlkoch** [comment]
Thanks for the review!

**Pull request merged**